addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4561,19 +4564,14 @@ export class OdooEditor extends EventTarget {
             }
         }
 
-        if (this.options.showEmptyElementHint) {
```

Comment:
@deso-odoo I believed that muted text for headings, lists, etc., was disabled on the website because it remained visible even when they weren't focused, resulting in the website layout displaying unnecessary placeholders. However, with this PR, muted text will now only appear when the tag is focused, effectively preventing issues such as the 'Heading 1' placeholder from being visible when the tag is not focused. @dmo-odoo What are your thoughts on it?

Reply:
I think I remember that @qsm-odoo or @rdeodoo didn't like those placeholders at all and wanted them removed from website, regardless of whether the tag was focused or not. They will likely chime in to confirm if I remember correctly or not.

Reply:
@dmo-odoo If we are speaking about the "Type "/" to ..." message, I like it. If we had problems with it at some point it was because it appeared at random places after random actions. If they appear at correct places consistently, they are of course good to keep.

Reply:
@qsm-odoo AFAIK we are talking about the hints like "Heading 1" when the heading is empty.
See: <img width="434" alt="Screenshot 2024-05-21 at 1 29 49 PM" src="https://github.com/odoo/odoo/assets/8427726/fb3216fe-5d5c-472a-9812-82a87ea451f2">

@deso-odoo This task should not affect the "Type / for commands" hints, right? I can't check the task because the Odoo server is under maintenance... ^^'

Reply:
Yes you are correct It does not affect "Type / for commands". The question is only for hints like Heading 1, Heading 2, etc...

Reply:
Thanks @qsm-odoo !

@deso-odoo It looks like @dhba-odoo remembered it better than we did after all. As long as the hint is only displayed on the element that has focus, it's fine. However, if we remove the `if` then the option serves no purpose anymore and could be removed. Two options here:

A. Remove the option entirely.

B. Keep the option and the condition in case we want to disable hints somewhere else down the line and simply remove the line in website that sets it to `false` so that hints are displayed in website.

If we had a PO I would ask them but since we don't, I'd rather do option B just in case.